android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/AuthenticationManager.java

```diff
@@ -269,35 +251,26 @@ public UserSessionInfo getUserSessionInfo() {
         // TODO: a way to distinguish if session is null because we haven't signed on, or if it
         // was invalidated
         UserSessionInfo session = apiClientProvider.getUserSessionInfoProvider()
```

Comment:
For more context for the curious, the UserSessionInfoProvider provides sessions for multiple signIns, and is more general than what the AuthenticationManager does. Currently, all the managers are synced to one signIn (registered upon calling authManager.signIn).

UserSessionInfoProvider itself is backed by a class that intercepts HTTP requests to extract UserSessionInfo from 412s, calls which return UserSessionInfo, and track signIn/signOut.